guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,1095 @@
++ const client = new Client({ intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES] });
+```
+
+### Structures#extend
+
+The concept of extendable Structures has been completely removed from Discord.js.
+
+For more information on why this decision was made and the recommended alternatives, refer to the [pull request](https://github.com/discordjs/discord.js/pull/6027).
+
+### Collectors
+
+All Collector related classes and methods (both create and await) now take a single object parameter which also includes the filter.
+
+```diff
+- const collector = message.createReactionCollector(filter, { time: 15000 });
++ const collector = message.createReactionCollector({ filter, time: 15000 });
+```
+
+### Naming conventions
+
+Some commonly used naming convention in Discord.js have been changed:
+
+#### Thing#thingId
+
+The casing of `thingID` properties has changed to `thingId`. This is a more-correct casing for the camelCase used by Discord.js as `Id` is an abbreviation, not an acronym.
+
+```diff
+- console.log(guild.ownerID);
++ console.log(guild.ownerId);
+
+- console.log(interaction.channelID);
++ console.log(interaction.channelId);
+```
+
+#### Client#message
+
+#### Client#interaction
+
+- `message` and `interaction` events have been renamed to `messageCreate` and `interactionCreate` respectively, to bring the library in line with Discord's naming conventions.
+
+Don't worry - the old names still work, but you'll receive a Deprecation Warning until you switch over.
+
+```diff
+- client.on("message", message => { ... });
++ client.on("messageCreate", message => { ... });
+
+- client.on("interaction", interaction => { ... });
++ client.on("interactionCreate", interaction => { ... });
+```
+
+### Snowflakes
+
+For TypeScript users, discord.js now enforces the `Snowflake` type, a stringified bigint, rather than allowing any string to be accepted.
+
+```diff
+interface Config {
+	prefix: string;
+- ownerId: string;
++ ownerId: Snowflake;
+}
+```
+
+### Allowed Mentions
+
+`clientOptions.disableMentions` has been removed and replaced with `clientOptions.allowedMentions`!
+
+The Discord API now allows bots much more granular control over mention parsing, down to the specific id.
+
+Refer to the [Discord API documentation](https://discord.com/developers/docs/resources/channel#allowed-mentions-object) for more information.
+
+```diff
+- const client = new Discord.Client({ disableMentions: 'everyone' });
++ const client = new Discord.Client({ allowedMentions: { parse: ['users', 'roles'], repliedUser: true } });
+```
+
+### Replies / Message#reply
+
+`Message.reply()` will no longer result in the bot prepending a user mention to the content, replacing the behavior with Discord's reply feature.
+
+`MessageOptions.reply` no longer takes a user id. It has been replaced with a `ReplyOptions` type, expecting `MessageOptions.reply.messageReference` as a Message id.
```

Comment:
The way you access to a typedef's property across this page is inconsistent, on line 124 you use `#`, while here you use `.`.